library/src/main/java/com/novoda/downloadmanager/lib/db/BetterCursor.java

```diff
@@ -0,0 +1,27 @@
+package com.novoda.downloadmanager.lib.db;
+
+import android.database.Cursor;
+
+public class BetterCursor extends CursorWrapper {
+
+    public static BetterCursor wrap(Cursor cursor) {
+        return new BetterCursor(cursor);
+    }
+
+    BetterCursor(Cursor cursor) {
+        super(cursor);
+    }
+
+    public int getInt(String columnName) {
+        return getInt(getColumnIndexOrThrow(columnName));
```

Comment:
I understand your sentiment here, trying to enforce the idea of throwing all the time, which is good! However anyone who normally uses cursor will be really confused by using this method ie.. `getInt` that does `getIntOrThrow`. It seems the throw is now implicit and you should always favor explicit code. This might be a bit of over engineering, lets keep it simple? The rest of the PR is ready to ship! :)